library/Mockery/Mock.php

```diff
@@ -724,7 +724,10 @@ public function mockery_returnValueForMethod($name)
             case 'int':    return 0;
             case 'float':  return 0.0;
             case 'bool':   return false;
-            case 'array':  return [];
+
+            case 'array':
+            case 'iterable':
```

Comment:
This is safe because in PHP less than 7.1, iterable will be parsed to \iterable by my new code.